config/initializers/ruby_stackoverflow.rb

```diff
@@ -1,5 +1,4 @@
 RubyStackoverflow.configure do|config|
-  config.client_key = ''
-  config.access_token = ''
+  config.client_key = ""
```

Comment:
I think this should take the values from environmental variables instead of having an empty string